presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -234,38 +236,20 @@ private DataTypeTest varcharDataTypeTest()
                 .addRoundTrip(varcharDataType(), "unbounded");
     }
 
-    @Test
```

Comment:
Actually it is against the good practice to keep separate test cases in separate tests.

> We usually test all aspects of a type within single test method.

We do it, and I am not very proud of it.

Reply:
I am guessing this comment is attached to `Merge test methods` commit.

I disagree, at least today. The test mapping test to be quite long, and having a way to reduce number of test methods -- without harming readability -- is useful and helps navigate the class.

This  is similar to how we group several test cases in here 
https://github.com/prestosql/presto/blob/d54e663db8d819ca4791ab61695f3d26cc0fccfb/presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java#L246-L272

I am happy we do not try to invent descriptive names for each of the test cases
(i like descriptive names, but i believe this is not really doable in a e consistent manner)

Anyway, I am dropping the  `Merge test methods` commit (and `Reorder test methods`) commits here.